fiftyone/core/models.py

```diff
@@ -182,6 +197,55 @@ def apply_model(
         )
 
 
+def apply_flash_model(
```

Comment:
Exposing this here is not necessary, since `apply_model()` itself accepts Flash models. As a matter of design taste, I'd recommend keeping Flash-specific code in `fiftyone.utils.flash`.